packages/components/src/components/hds/layout/grid/index.ts

```diff
@@ -115,10 +134,40 @@ export default class HdsLayoutGrid extends Component<HdsLayoutGridSignature> {
     if (this.args.columnMinWidth) {
       inlineStyles['--hds-layout-grid-column-min-width'] =
         this.args.columnMinWidth;
+      inlineStyles['--hds-layout-grid-column-fill-type'] = 'auto-fit';
```

Comment:
[question] usually we use CSS variables in JS to pass down to the CSS dynamic values set by the consumers; in this case, we already know the two possible values (`auto-fit` and `auto-fill`) and the conditions for them, so could we simply use CSS classes assigned in the `get classNames()` getter below, using this same logic?

Reply:
I replaced it with using a classname as nothing seemed to rely on needing an inline style (I think this might have been left over from an earlier implementation before I fixed an issue with nested Grids inheriting styles incorrectly)